install_config/aggregate_logging.adoc

```diff
@@ -916,63 +904,15 @@ As part of the OpenShift cluster installation, it is recommended that you add th
 Fluentd node selector to the list of persisted
 xref:../install_config/install/advanced_install.adoc#configuring-node-host-labels[node labels].
 
-Fluentd uses `journald` as the system log source. These are log
-messages from the operating system, Docker, and OpenShift. For
-container logs, Fluentd determines which log driver Docker is using,
-`json-file` or `journald`, and automatically reads the logs from that source.
-
-// [[fluentd-use-journald]]
-// *Having Fluentd Use the Systemd Journal as the Log Source*
-
-// By default, Fluentd reads from `journald` and
-// *_/var/log/containers/<container>.log_* for system logs and container logs,
-// respectively. You can instead use the systemd journal as the log source. There
-// are three inventory parameters available:
-
-// [cols="3,7",options="header"]
-// |===
-// |Parameter
-// |Description
-
-// | `journal-source`
-// |The default is empty, so that when using the systemd journal, Fluentd first looks for
-// *_/var/log/journal_*, and if that is not available, uses *_/run/log/journal_*
-// as the journal source. You can specify `journal-source` with an explicit
-// journal path. For example, if you want Fluentd to always read logs
-// from the transient in-memory journal, set `journal-source`=*_/run/log/journal_*.
-
-// | `openshift_logging_journal_read_from_head`
-// |If this setting is `false`, Fluentd starts reading from the end of the journal,
-// ignoring historical logs. If this setting is `true`, Fluentd starts reading
-// logs from the beginning of the journal.
-// |===
+With Fluentd, it is recommended to use `json-file` as the system log source.
+These are log messages from the operating system, Docker, and OpenShift. For
```

Comment:
Does it makes sense to keep "system log source"?  This seems like it might confuse folks.

The docker log driver is only about how container logs are delivered, right?

Then the next sentence seems like it needs revision as "These" seems out of place, but also that "operating system" and "Docker", and "OpenShift" logs don't necessarily go through `json-file` log driver.

Perhaps this make sense to other folks?

Reply:
If it makes sense to remove the log source sentence ("Fluentd uses `journald` as the system log source."), I can. Is it important for readers to know that the log source is `journald`? And is the log source still `journald` if the log driver is `json-file`? 

I've looked into this a bit and can't seem to find a definitive answer.

Reply:
How about just this:
```
Fluentd uses journald as the system log source.  These are log messages from the operating system, the container runtime, and OpenShift.
```

Reply:
Thanks, @richm! Updating now.